### DISCUSSION TITLE
请教下如何更改菜单主题为白色背景，字体黑色

### DISCUSSION THREAD

**Discussion by IvesQing** [vbenjs/vue-vben-admin#3154] [General]
如图所示，想要把菜单主题改为白色背景，字体黑色，框架版本2.10，折腾了很久，弄不出来

![image](https://github.com/vbenjs/vue-vben-admin/assets/26543986/823496ce-8904-4428-9136-3c0fbe504655)

**Comment by brosegu** [comment]
菜单背景色projectSetting.menuSetting.bgColor修改无效
